github/pulls.go

```diff
@@ -251,12 +251,14 @@ type PullRequestMergeResult struct {
 
 // PullRequestOptions lets you define how a pull request will be merged.
 type PullRequestOptions struct {
-	Squash bool
+	Squash      bool
+	MergeMethod string
```

Comment:
Can you please document this field?
e.g.: `Merge method to use. Possible values are: "merge", "squash", "rebase". Default is "merge".`